src/cljs/athens/views/right_sidebar/shared.cljs

```diff
@@ -111,5 +111,5 @@
                :children   (r/as-element (cond
                                            (= type "graph") [graph/page name]
                                            (= type "page") [node-page/page eid]
-                                           :else [block-page/page eid]))}))
+                                           :else [blocks/page eid]))}))
```

Comment:
We should check for BlockTypeProtocol availability and use it here.

Reply:
You mean this `renderer-k                 (block-type-dispatcher/block-type->protocol-k block-type ff)`? 
I am using it in the block-page-el.